src/main/java/org/cyclops/integrateddynamics/core/network/PartNetworkElement.java

```diff
@@ -64,6 +64,11 @@ public int getPriority() {
     }
 
     @Override
+    public int getChannel() {
+        return hasPartState() ? part.getChannel(getPartState()) : 0;
```

Comment:
For consistency, use `IChanneledNetwork.DEFAULT_CHANNEL` here as well.

Reply:
Good catch.